admin_ui/backend/api/tools.py

```
@@ -506,17 +506,52 @@ def _extract_json_paths(obj: Any, prefix: str = "") -> List[Dict[str, str]]:
+    """
+    try:
+        from settings import ENV_PATH
+        if not os.path.exists(ENV_PATH):
+            return None
+        from dotenv import dotenv_values
+        raw = dotenv_values(ENV_PATH)
+        val = raw.get(name)
+        if val is None:
+            return None
+        return str(val)
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)